_layouts/default.html

```diff
@@ -8,10 +8,11 @@
   <a href="{{ '/' | relative_url }}" class="title">CODEVID-19</a>
   <nav>
     <ul>
-      <li><a href="{{ '/' | relative_url }}">Home</a></li>
-      <li><a href="{{ '/policies/rules.html' | relative_url }}">Rules</a></li>
-      <li><a href="{{ '/policies/submissions.html' | relative_url }}">Submit Your Project</a></li>
```

Comment:
I'm not comfortable dropping `Submit Your Project` and `FAQ` from the menu simple to save space. It seems like we're compromizing function for form.

Looking at what other pages do, perhaps it's more reasonable to try a hamburger menu or similar. I don't think that simple removing them is reasonable though.

Reply:
Reverted